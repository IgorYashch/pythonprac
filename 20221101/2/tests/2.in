a = Triangle((-1, 0), (0, 10), (3, -5))

print(round(abs(a), 3))
print(f"Triangle is {'not' if a else ''} empty")

b = Triangle((0, 0), (1, 0), (1, 1))

print(b in a)
print(b < a)
print(b & a)