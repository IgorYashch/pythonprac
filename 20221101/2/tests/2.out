22.5
Triangle is not empty
True
True
True
